python/tvm/relay/op/nn/nn.py

```diff
@@ -1994,7 +1994,7 @@ def batch_matmul(x, y):
 
 
 # pylint: disable=no-else-return,inconsistent-return-statements
-def sparse_dense(data, weight, sparse_data=False):
+def sparse_dense(data, weight, sparse_lhs=False):
     r"""
     Computes the matrix multiplication of `data` and `weight`, where `data` is
```

Comment:
Please clarify what operation is occurring here. S * D^T if sparse_lhs, D * S^T otherwise.

Reply:
Should we change the name data & weight to dense_mat & sparse_mat ?

Reply:
Yeah, that might make more sense.